bin/crew

```diff
@@ -1404,6 +1410,9 @@ def install
     # perform pre-install process
     pre_install dest_dir
 
+    # perform install_multiple process
+    install_multiple_package dest_dir
+
     # perform install process
     install_package dest_dir
```

Comment:
Is `install_multiple_package` an either/or for `install_package`?  If so, do we need an if...else...end?

Reply:
We don't have an if/then for preinstall...

Reply:
What I'm asking is do we need to run both `install_multiple_package` and `install_package` or just one or the other?

Reply:
Right now I'm working on using both.

The install_multiple modifies filelist and dlist, and install_package then installs all files in (the resultant modified or unmodified) filelist and dlist.

That way the existing install algorithm stays largely intact.

Reply:
Actually, do you even need a separate function?  Can't that go in preinstall instead?

Reply:
You're right.